quartz/plugins/emitters/ogImage.tsx

```diff
@@ -73,6 +76,23 @@ async function processOgImage(
 ) {
   const cfg = ctx.cfg.configuration
   const slug = fileData.slug!
+  const outputPath = path.join(ctx.argv.output, `${slug}-og-image.webp`)
+  
+  // Skip regeneration if OG image exists and is newer than source
+  if (SKIP_EXISTING) {
```

Comment:
Please add a comment explaining the full caching strategy and pointing to the file that contains our corresponding build time caching mechanism